helix-tui/src/backend/crossterm.rs

```diff
@@ -23,13 +23,32 @@ use std::{
     fmt,
     io::{self, Write},
 };
+use termini::TermInfo;
 
 fn term_program() -> Option<String> {
-    std::env::var("TERM_PROGRAM").ok()
+    // Some terminals don't set $TERM_PROGRAM
+    match std::env::var("TERM_PROGRAM") {
+        Err(_) => std::env::var("TERM").ok(),
+        Ok(term_program) => Some(term_program),
+    }
 }
 fn vte_version() -> Option<usize> {
     std::env::var("VTE_VERSION").ok()?.parse().ok()
 }
+fn reset_cursor_approach(terminfo: TermInfo) -> String {
+    let Some(termini::Value::Utf8String(se_str)) = terminfo.extended_cap("Se") else {
+        return "".to_string();
+    };
+
+    format!(
+        "{}{:?}{}",
```

Comment:
`"{:?}"` is wrong here, that is why the cursor resetting was working in tmux for me. I accidentally left the `"{:?}"` while trying to figure out the types because I'm a rust noob xD